### PR TITLE
.sqrt() expression

### PR DESCRIPTION
**Pull request by dmaymay**
Working .sqrt() function. Casting integers to appropriate float types :)

### CODE REVIEW

File: src/daft-dsl/src/functions/numeric/sqrt.rs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            DataType::Int16 => DataType::Float32,
+            DataType::UInt8 => DataType::Float32,
+            DataType::UInt16 => DataType::Float32,
+            DataType::Int32 => DataType::Float64,
+            DataType::Int64 => DataType::Float64,
+            DataType::UInt32 => DataType::Float64,
+            DataType::UInt64 => DataType::Float64,
+            DataType::Float32 => DataType::Float32,
+            DataType::Float64 => DataType::Float64,
+            dt if dt.is_numeric() => DataType::Float64,
```
Comment (colin-ho) [inline src/daft-dsl/src/functions/numeric/sqrt.rs:36]: Do you need this `is_numeric` check still? I believe you've covered all the cases above this line already

**Review by colin-ho — COMMENTED** [review]
Awesome work @dmaymay ! Just a few nits

File: src/daft-dsl/src/functions/numeric/sqrt.rs (hunk 1 above)
Comment (dmaymay) [reply src/daft-dsl/src/functions/numeric/sqrt.rs:36]: Nope don't think its needed anymore

**Comment by colin-ho** [comment]
Hey @dmaymay ! This PR looks ready to merge, if you could just help remove this [one line here](https://github.com/Eventual-Inc/Daft/pull/2180/files#diff-49de40913435c5e330d259b673362472c8dac5b4721c32636986912552813275R36) that's no longer needed that'd be great. Also seems like there's a merge conflict, I checked it out and it should just be a simple one line change :)

**Comment by dmaymay** [comment]
@colin-ho We good to merge?

**Comment by colin-ho** [comment]
> @colin-ho We good to merge?

Yep! Just did a rebase to inspect the file changes a lil easier, but all good and merged now, thanks @dmaymay 😆

**Comment by colin-ho** [comment]
Also congrats on first PR!!

**Comment by jaychia** [comment]
Congrats on your first PR 🤓 @dmaymay

**Comment by samster25** [comment]
great job @dmaymay!

**Pull request merged**